sdk/storage/test/test_base.cpp

```diff
@@ -1,15 +1,129 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
+#ifdef _WIN32
```

Comment:
Use `_MSC_VER` instead here, please. I think we can still use other compilers on Windows and the macro was not useful for them.

Reply:
fixed